tests/get-price.test.ts

```diff
@@ -78,15 +78,16 @@ describe("example tests", () => {
 
 describe("mint-sbtc demo", () => {
   it("can get tx costs", () => {
-    const blockHeight = 1000;
+    const burnBlockHeight = 881000;
```

Comment:
there's no stacks block associated to burn block 1000.

IT was working with clarinet <= 2.15.0 because it was using mock bitcoin data